lib/mandrill_dm/message.rb

```diff
@@ -61,7 +61,11 @@ def headers
     end
 
     def html
-      mail.html_part ? mail.html_part.body.decoded : mail.body.decoded
+      if mail.html_part
+        mail.html_part.body.decoded
+      elsif !mail.text?
+        mail.body.decoded
+      end
     end
```

Comment:
Small style-nit: I'm not a fan of this conditional style with an implicit `nil` return when neither matches. I think I'd rather see this using guard clauses and an explicit `nil` return:

```
def html
  return mail.html_part.body.decoded if mail.html_part
  return mail.body.decoded if mail.text?
  nil
end
```